### ISSUE TITLE
One-way collision platform

### ISSUE THREAD

**Issue by gcivicodev** [vnen/godot-tiled-importer#37]
Hi @vnen 

There is a way to set one-way collision platform using the importer?

**Comment by vnen** [comment]
No, Godot 2 does not support one-way collision on tiles. It could happen on objects in an object layer, but it would depend on custom properties.

**Comment by Lucrecious** [comment]
Now that one-way collision is supported on tiles, would it be possible to add this in?

**Comment by vnen** [comment]
Yes, it could be added. Probably relying on custom properties as I said before.

**Comment by Lucrecious** [comment]
It might be cool to reserve properties for the "tile properties" and instead parse the Tile Type.

i.e. the type could be "kinematic, one_way" which means the tile's collision is kinematic and one-way collision.

Just a suggestion - great addon btw :)

**Comment by salvob41** [comment]
I am very much interested to this feature. This is the actual workaround that me and my team implemented to make Tiled work with one way collision, in case someone needs it.

https://notapixelstudio.wordpress.com/2018/03/10/one-way-collision-tile-with-tiled-and-godot/

TLDR; 
- Use automapping to map a layer (we called "jumptrhough", for example) 
- Get the tiles from gd script with `map.get_node('jumpthrough')`
- use the gd API: `tile_set.tile_set_shape_one_way(tile_id, #shape_id, true)` (where _tile_id_ is the id property from Tiled plus 1 (I don't know why Tiled starts the index from 0, while the plugin starts from 1) and _shape_id_ is the index of the collisionShape2D for that tile)

Thank you for the good work

**Comment by vnen** [comment]
Add as suggested by @Lucrecious: set the object type to `one-way` to make this happens (works both on objects in a tile and in an object layer).

Note that Godot 2 got support for one-way collision in tiles, but I'm not willing to support the 1.x version of this plugin anymore (apart from bugfixes). I'll merge PRs if anyone wants to add features to it.
